### ISSUE TITLE
Increase Font Size button is at end of tab order but is visually at top

### ISSUE THREAD

**Issue by jhung** [pressbooks/pressbooks-book#172]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all plugins except Pressbooks deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/), the latest version of [Pressbooks Book](http://github.com/pressbooks/pressbooks-book/releases/latest/), the latest version of [this book theme](http://github.com/pressbooks/pressbooks-book/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing the issue while using this book theme?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks-book) to see if your bug or enhancement is already listed?

### Description

The tab order for Increase Font Size comes after the Footer, but it appears visually at the top of the document. Tab order should (typically) follow local reading direction, so the Increase Font Size should probably occur after Skip to Content link.

### Steps to Reproduce

1. Load a book
2. Repeatedly press tab to move focus through the content page

**Expected behavior:**

Increase Font Size tab order should be near the beginning (possibly after the Skip to Content link).

**Actual behavior:** 

Increase Font Size is the last tabbable item on the page - after the footer links.
